File: egym_ui_mobile/lib/app/models/service.dart
class Service {
  final String title;
  final String desc;
  final String image;

  Service({
    required this.title,
    required this.desc,
    required this.image,
  });

  factory Service.fromJson(Map<String, dynamic> json) {
    return Service(
      title: json["title"],
      desc: json["descritption"],
      image: json["picture"],
    );
  }
}

File: eGym.Domain/Service.cs
using System;
namespace eGym.Domain;

public class Service
{
	public int ServiceId { get; set; }
	public string Descritption { get; set; } = string.Empty;
	public string Title { get; set; } = string.Empty;
	public string Picture { get; set; } = string.Empty;
}